### ISSUE TITLE
Problem for your squeezeSeg-Pytorch Repo

### ISSUE THREAD

**Issue by BenjaminYoung29** [mfxox/ILCC#17]
Hi. Sorry to leave a issue here about your another repository. I've try to run your code and fix some little problems and the result wasn't pleasant. Loss won't decrease a bit while training. And the iou are all nearly 0. Have you get a good result on it yet? I can't find any problem in the train.py or sequeezeSeg.py. Do you have any idea?
--------
I deleted the bf and crf layers and the loss began to devrease. however the iou is really poor(0.03 for car and 0.0001 for cyclist) and output in the tensorboard is really bad.
